card/build.gradle

```diff
@@ -52,9 +54,4 @@ dependencies {
 
     // Dependencies
     implementation "com.google.android.material:material:$material_version"
-}
-
-// This sharedTasks.gradle script is applied at the end of this build.gradle script,
-// since javadocs.gradle script is dependent on android.compileSdkVersion property,
-// which is set on the android block above.
-apply from: "../config/gradle/sharedTasks.gradle"
```

Comment:
Minor, but I see this "new line at end of file" on klint as well, now github is giving a warning as well.